writeengine/wrapper/writeengine.cpp

```diff
@@ -6119,6 +6578,42 @@ void WriteEngineWrapper::AddDictToList(const TxnID txnid,
 
 }
 
+// Get CPInfo for given starting LBID and column description structure.
+int WriteEngineWrapper::GetLBIDRange(const BRM::LBID_t startingLBID, const ColStruct& colStruct, BRM::CPInfo& cpInfo)
+{
+    int rtn;
+    BRM::CPMaxMin maxMin;
+    bool isBinary = colStruct.colWidth > 8;
```

Comment:
Plz use datatypes::MAXLEGACYWIDTH constant.

Reply:
Moved to we_brm.h, ExtCPInfo::isBinaryColumn().